apps/frontend/package.json

```diff
@@ -51,5 +50,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@vitejs/plugin-react": "^3.1.0",
```

Comment:
Replace with:

https://github.com/vitejs/vite-plugin-react-swc

And read about about "swc vs babel"